.current-version

```diff
@@ -0,0 +1 @@
+3.2.0
```

Comment:
I'm not too crazy about keeping this extra piece of state around. Although it's wacky, i'd prefer to use some bash to pull the version out of the `.bumpversion.cfg` file. You could modify the `bump` script similarly to this: https://github.com/recurly/recurly-client-node/blob/master/scripts/bump#L7

Reply:
Yeah, I agree. I'll put something else together with changelog generation